File: native/android/src/main/java/com/nozbe/watermelondb/Queries.java
package com.nozbe.watermelondb;

public class Queries {
    public static final String select_local_storage = "select value from local_storage where key = ?";
    public static final String select_tables = "select * from sqlite_master where type='table'";
    public static String dropTable(String table) {
        return "drop table if exists `" + table + "`";
    }
}

File: native/android/src/main/java/com/nozbe/watermelondb/Queries.kt
package com.nozbe.watermelondb

object Queries {
    const val select_local_storage = "select value from local_storage where key = ?"
    const val select_tables = "select * from sqlite_master where type='table'"
    fun dropTable(table: String) = "drop table if exists `$table`"
}
